src/Resources/Transactions.php

```diff
@@ -52,4 +52,43 @@ public function refresh(string $access_token): object
 			$this->paramsWithClientCredentials($params)
 		);
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Get transactions for a Plaid Item since the last sync.
+	 * @see https://plaid.com/docs/api/products/transactions/#transactionssync
+	 *
+	 * @param string $access_token
+	 * @param string $cursor The "cursor" (or page identifier) provided from the
+	 *               last sync request. Do not provide cursor for the first-ever
+	 *               sync request for an item.
+	 * @param int $count Number of transactions per response (or page size)
+	 * @param array<string,mixed> $options
+	 * @throws PlaidRequestException
+	 * @return object
+	 */
+	public function sync(string $access_token, ?string $cursor = null, ?int $count = null, ?array $options = null): object
```

Comment:
All other methods define the $options array as `array $options = []`. Can we be consistent with those?